src/Definition/ClassDefinition.php

```diff
@@ -58,12 +58,12 @@ class ClassDefinition extends ParentDefinition
     /**
      * @var string|null
      */
-    protected $extendsClass;
+    protected $extendsClass = null;
```

Comment:
It was null before (since we don't set it except when we call setExtendsClass which is not guaranteed to happen). So I just made it explicit. I thought it would be easier to understand why we are checking for !== null at other points in the code.